Makefile

```diff
@@ -213,17 +227,76 @@ update-codegen:
 	$(call run-informer-gen,)
 .PHONY: update-codegen
 
+# $1 - target file
+define generate-crd
+	$(CONTROLLER_GEN) crd paths="$(GO_PACKAGES)" output:crd:stdout > '$1'
+endef
+
+update-crd:
+	$(call generate-crd,pkg/api/scylla/v1/scylla.scylladb.com_scyllaclusters.yaml)
+.PHONY: update-crd
+
+verify-crd: tmp_file :=$(shell mktemp)
+verify-crd:
+	$(call generate-crd,$(tmp_file))
+	$(diff) '$(tmp_file)' pkg/api/scylla/v1/scylla.scylladb.com_scyllaclusters.yaml || (echo 'CRD was not regenerated' && false)
+.PHONY: verify-crd
+
+# $1 - target dir
+# $2 - source dir
+define copy-amend-manifests
+	mkdir -p '$1' && \
+	cp '$2'/*.yaml '$1' && \
+	yq eval '.spec.cpuset=false | .spec.datacenter.racks[] = (.spec.datacenter.racks[] | .resources.limits.cpu="200m" | .resources.limits.memory="200Mi" | .resources.requests.cpu="10m" | .resources.requests.memory="100Mi" )' -i '$1'/10_scylla_scyllacluster.yaml
+endef
+
+update-dev-deploy:
```

Comment:
for bash completion and hierarchy it might be better reorder it into `update-deploy-dev`